content/200-orm/050-overview/500-databases/200-database-drivers.mdx

```diff
@@ -156,3 +156,24 @@ import { PrismaClient } from 'db'
 
 const client = new PrismaClient()
 ```
+
+### Driver adapters and specific frameworks
+
+#### Nuxt
+
+Using a driver adapter with [Nuxt](https://nuxt.com/) to deploy to an edge function environment does not work out of the box, but adding the `nitro.experimental.wasm` configuration option fixes that:
```

Comment:
Nit: let's specify that we're talking about Nuxt 3?

Reply:
```suggestion
Using a driver adapter with [Nuxt 3](https://nuxt.com/) to deploy to an edge function environment does not work out of the box, but adding the `nitro.experimental.wasm` configuration option fixes that:
```

Reply:
Let's omit the Nuxt version entirely from the docs. Perhaps we can simply rephrase the first sentence with

```suggestion
Using a driver adapter with [Nuxt](https://nuxt.com/) and [Nitro](https://nuxt.com/docs/api/kit/nitro) to deploy to an edge function environment does not work out of the box, but adding the `nitro.experimental.wasm` configuration option fixes that:
```

Reply:
How does Nitro play into this? Is that just used in the newer versions, where our workaround implies?

Reply:
Moved to https://github.com/prisma/docs/pull/6054#pullrequestreview-2113102443